front/app/containers/App/index.tsx

```diff
@@ -505,7 +505,7 @@ class App extends PureComponent<Props, State> {
                   >
                     <ErrorBoundary>{children}</ErrorBoundary>
                     <HasPermission.No>
-                      <ForbiddenRoute />
+                      <Redirect method="push" path="/" />
```

Comment:
There is a usePermission hook that can be used instead of this `HasPermission` component, if this is the reason `Redirect` needs to be a component. But I'm not 100% sure they have the exact same behaviour so just pointing it our for info. I'm also wondering if we could use `redirect` directly from `react-router-dom`?

Reply:
Yeah I didn't dare to touch this part, but you're right- `react-router-dom` has a `Navigate` component which I think we can use here. I'll test if that works instead